content/blog/2021-11-30-international-rse-survey.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Take the 2021 international RSE survey: help us improve the RSE community"
+
+[extra]
+
+author = "Patric Holmvall, Richard Darst" 
++++
+
+
+## Take the survey
+
+Do you develop research codes, or generally work with numerics or data analysis? Perhaps you even feel more connected to the technology side: programming, software, data, etc., rather than the writing of papers? Although this work is crucial to the research, the modern research environment seldom fully recognizes this work, in contrast to other methodology development. Take part in the [international Research Software Engineer (RSE) survey](https://softwaresaved.limequery.com/386272?lang=en) to help us understand how we can improve the situation. Your current position or job title does not matter, only that you feel connected to software, data, computation, or similar technology.
+
+Link: <https://softwaresaved.limequery.com/386272?lang=en>
+
+
+## What/who is an "RSE"?
+
+A ["Research Software Engineer"](https://www.software.ac.uk/blog/2016-08-17-not-so-brief-history-research-software-engineers-0) (RSE) is a person who combines the mindset of a researcher with the skills of a software developer.  We interpret this inclusively, for people who might consider themselves data engineers, system specialists, and so on. For decades, research has been dependent on roles such as this, but there is very little career support or recognition for this role. The Research Software Engineer movement aims to improve this situation by representation and advocating for these people and their work.
+
+
+## RSEs in the Nordics and Baltics
+
+[Nordic-RSE](https://nordic-rse.org/events/) is the Nordic+Baltics organization advocating for RSEs. Founded in 2017 and registered in Finland in 2021, it provides a [community](https://coderefinery.zulipchat.com), [periodic seminars](https://nordic-rse.org/events/seminar-series/), and in the future [other events](https://nordic-rse.org/events/) which can connect RSEs to like-minded individuals to promote career development. It will also advocate for the value of RSE services within the research process.
```

Comment:
```suggestion
[Nordic-RSE](https://nordic-rse.org/) is the Nordic+Baltics organization advocating for RSEs. Founded in 2017 and registered in Finland in 2021, it provides a [community](https://coderefinery.zulipchat.com), [periodic seminars](https://nordic-rse.org/events/seminar-series/), and in the future [other events](https://nordic-rse.org/events/) which can connect RSEs to like-minded individuals to promote career development. It will also advocate for the value of RSE services within the research process.
```